libHVCAENx527App/Db/HVCAENx527ch.tpl

```diff
@@ -2,299 +2,307 @@
 # Copyright Canadian Light Source, Inc.  All rights reserved.
 #    - see licence.txt and licence_CAEN.txt for limitations on use.
 #
-# The following are for setting channel parameters
+
+#
+# define ai before ao as the ao init record reads a default value
+# and this will fail if "epics_enabled" is not set, which is done
+# by the ai init
+#
+
+#
+# The following are for monitoring channel parameters (e.g. feedback)
 # in user space
 #
-record( stringout, "$(PSNAME):$(CHANNUM):name")
+record( stringin, "$(PSNAME):$(CHANNUM):name:fbk")
```

Comment:
Since we're modifying this DB extensively already, should we take the opportunity to make it fit our standards (e.g. `VALUE`, `VALUE:SP`, `VALUE:SP:RBV`)?

Reply:
I didn't think we were changing it so much as to make that worth doing, but we can discuss.